.travis.yml

```diff
@@ -14,11 +16,16 @@ before_install:
   - cp config/secrets.travis.yml config/secrets.yml
   - git submodule init
   - git submodule update --depth=10 -- red5
+  - rvm use 2.3.3@global
```

Comment:
Any reason for this? You are specifying this using travis' syntax `rvm: -2.3.3` at the bottom of this file?